node/lib/package/PackageInfo.ts

```diff
@@ -181,3 +184,25 @@ export class PackageTranslatorInfo {
      */
     public readonly onboardingProperties: { [propertyName: string]: string };
 }
+
+export class PackageOnboardingInfo
+{
+    public readonly onboardingId: string;
+    public readonly schemas: string[];
```

Comment:
@tjaffri, @jeanpa @arjun-msft to comment on this

---
In reply to: [75262678](https://github.com/openT2T/opent2t/pull/22#discussion_r75262678) [](ancestors = 75262678)